app/views/orders/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="row">
+<div class="col-md-8 col-md-offset-2">
+
+  <h2><center>Shopping Cart</center></h2>
+
+  <% unless session[:order_id].nil? %>
+    <h3><center>You have <%= item_total(@order) %> items in your cart</h3>
+    <h3><center>Subtotal <%= number_to_currency(@total) %></h3>
+ <% end %>
+</div>
+
+  <div class="shopping-cart col-md-10 col-md-offset-1">
+    <%= render "shopping_cart" %>
+  </div>
+
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4 col-md-offset-7">
+
+        <% if  @order_items.any? { |order_item| order_item.quantity > order_item.product.stock } %>
```

Comment:
Tried to, kept getting an unexpected end error. I think I struggle on how to move it into a helper or the model when it is an instance variable.